addons/package_hierarchy/models/stock_quant_package.py

```diff
@@ -129,3 +132,19 @@ def action_toggle_processed(self):
             else:
                 for ml in move_lines:
                     ml.qty_done = 0
+
+    def action_view_picking(self):
+        """
+        Override method to be able to see package transfers from a parent package too.
+        Default filter will be confirmed picks:
+          state in ('confirmed', 'waiting', 'assigned')
+        """
+        action = self.env.ref("stock.action_picking_tree_all").read()[0]
+        packages = self.search([("id", "child_of", self.ids), ("package_id", "!=", False)])
+        domain = [
+            "|", ("result_package_id", "in", packages.ids), ("package_id", "in", packages.ids)
+        ]
+        pickings = self.env["stock.move.line"].search(domain).mapped("picking_id")
```

Comment:
Nit: Can we put `["stock.move.line"]` into a variable at the top of the method?

Reply:
It is used only once, and it was in core method too. But yes can do that